examples/guides/se-restful-webservice/README.adoc

```diff
@@ -70,135 +73,78 @@ See the <<downloading,downloading>> section for instructions.
 
 == Develop your application
 
-=== Create a new Maven project
-You can create your Maven project in these ways:
+=== Generate the Maven project using the Helidon archetype
+Helidon provides a Maven archetype you can use to create a new Helidon project that
+includes sample code.
 
-* use your IDE to create a new Java Maven project, or
-* run the standard Maven archetype to create a new Java project using this command:
+1. `cd` to a directory that is not already a Maven project.
+2. Run this command:
 +
-[source,bash]
-mvn archetype:generate -DarchetypeGroupId=org.apache.maven.archetypes -DarchetypeArtifactId=maven-archetype-quickstart -DarchetypeVersion=1.3
+--
+[source,bash,subs="attributes+"]
+.Creating a new Helidon SE project
+----
+mvn archetype:generate -DinteractiveMode=false \
+    -DarchetypeGroupId=io.helidon.archetypes \
+    -DarchetypeArtifactId=helidon-quickstart-se \
+    -DarchetypeVersion={helidon-version} \
+    -DgroupId=io.helidon.guides \
+    -DartifactId={artifact-id} \
+    -Dpackage=io.helidon.guides.se.restfulwebservice
+----
+
+Running the archetype this way creates a subdirectory `{artifact-id}`
+(using the `artifactId` setting from the archetype invocation) that contains a new
+Maven project for a Helidon service.
+--
+
+=== Browse the generated source
+
+==== `pom.xml`
+The input you provided to the archetype determines the project's Maven
+coordinates:
+[source,xml,indent=0]
+// _include::0-0:{pom}[tag=coordinates]
+----
+    <artifactId>se-restful-webservice</artifactId>
+----
 
-=== Update your `pom.xml`
-Add the properties and dependencies listed here if they don't exist in your `pom.xml`:
+Also in the `<dependency-management>` section notice the dependency on the Helidon bom pom:
```

Comment:
In the `<dependency-management>` section, note the dependency on the 'helidon-bom` POM:

Reply:
Fixed in PR #364 which addresses the SE tutorial.